post_installation_configuration/machine-configuration-tasks.adoc

```diff
@@ -0,0 +1,50 @@
+:context: post-install-machine-configuration-tasks
+[id="post-install-machine-configuration-tasks"]
+= Post-installation machine configuration tasks
+include::modules/common-attributes.adoc[]
+toc::[]
+
+There are times when you need to make direct changes to the
+operating systems running on {product-title} nodes. This can include
+changing settings for network time service, adding kernel arguments,
+or configuring journaling in a specific way.
+
+Aside from a few specialized features, most direct changes to
+operating systems on {product-title} nodes can be done by creating
+what are referred to as MachineConfig objects that are managed by the
+Machine Config Operator.
+
+Tasks in this section describe how to use features of
+the Machine Config Operator to configure operating system
+features on {product-title} nodes.
+
+[id="understanding-the-machine-config-operator"]
+== Understanding the Machine Config Operator
+
+include::modules/machine-config-operator.adoc[leveloffset=+2]
+include::modules/machine-config-overview.adoc[leveloffset=+2]
+include::modules/checking-mco-status.adoc[leveloffset=+2]
+
+[id="using-machineconfigs-to-change-machines"]
+== Using MachineConfigs to configure machines
+Tasks in this section let you create MachineConfig objects to modify files, systemd unit files, and other operating system features running on {product-title} nodes. For more ideas on working with MachineConfigs, see KCS content related to link:https://access.redhat.com/solutions/5307301[changing MTU network settings], link:https://access.redhat.com/solutions/5096731[adding] or
+link:https://access.redhat.com/solutions/4510281[updating] SSH authorized keys, , link:https://access.redhat.com/solutions/4518671[replacing DNS nameservers], link:https://access.redhat.com/verify-images-ocp4[verifying image signatures], link:https://access.redhat.com/solutions/4727321[enabling SCTP], and link:https://access.redhat.com/solutions/5170251[configuring iSCSI initiatornames] for {product-title}.
+
+include::modules/installation-special-config-crony.adoc[leveloffset=+2]
+include::modules/nodes-nodes-kernel-arguments.adoc[leveloffset=+2]
+include::modules/nodes-nodes-rtkernel-arguments.adoc[leveloffset=+2]
+include::modules/machineconfig-modify-journald.adoc[leveloffset=+2]
+include::modules/machineconfig-modify-registry.adoc[leveloffset=+2]
+include::modules/rhcos-add-extensions.adoc[leveloffset=+2]
+
+Use "Configuring crony time service" as a model for how to go about adding
```

Comment:
Use **the** "Configuring crony time service" **section**

Reply:
Done.